cogboard-webapp/src/helpers/index.js

```diff
@@ -41,3 +43,17 @@ export const parseYupErrors = errors => {
 export const trimLeadingZeros = inputValue => String(parseInt(inputValue));
 
 export const hasError = error => error !== undefined;
+
+export const postWidgetContentUpdate = (data = {}) => {
+  const postConfig = {
+    method: 'POST',
+    body: JSON.stringify(data),
+    headers: {
+      'Content-Type': 'application/json'
+    }
+  };
+
+  return fetch('/api/widget/contentUpdate', postConfig)
+    .then(checkResponseStatus)
+    .then(response => response.json());
+};
```

Comment:
Should probably move to `utils`. Which brings me to the question - why do we have `utils` and `helpers`? As far as I can tell, there is little to no difference, so both directories could potentially get merged.

Reply:
Waiting for Marek's input on this one

Reply:
Agreed on and modified per mutual observations.